PCbuild/pyproject.props

```
@@ -18,6 +18,7 @@
     <SupportSigning Condition="'$(SupportSigning)' == ''">true</SupportSigning>
     <SupportSigning Condition="'$(Configuration)' == 'Debug'">false</SupportSigning>
     <SupportSigning Condition="'$(ConfigurationType)' == 'StaticLibrary'">false</SupportSigning>
+    <LinkIncremental Condition="$(Configuration) == 'Release' or $(Configuration) == 'PGInstrument' or $(Configuration) == 'PGUpdate'">false</LinkIncremental>
```

```suggestion
    <LinkIncremental Condition="$(Configuration) != 'Debug'">false</LinkIncremental>
```

We don't have any other relevant configurations.

---

You're right that is more concise.  Will change.